pkg/ttnpb/udp/translation.go

```diff
@@ -220,8 +220,10 @@ func convertUplink(rx RxPacket, md UpstreamMetadata) (*ttnpb.UplinkMessage, erro
 	}
 
 	switch up.Settings.DataRate.Modulation.(type) {
```

Comment:
You can do `mod := up.Setings...` in the switch and then you can edit the coding rate via `mod.Lora.CodingRate` / `mod.Lrfhss.CodingRate`.